packages/@o3r/dynamic-content/src/services/dynamic-content/dynamic-content.service.ts

```diff
@@ -27,21 +27,11 @@ export class DynamicContentService {
     return !assetPath ? '' : assetPath.replace(/^\//, '');
   }
 
-  /**
```

Comment:
it was not mandatory but why removing it?

Reply:
It's a copy paste that is not accurate at all since it doesn't explain the difference content VS media, I prefer adding an accurate one on both public methods, and delete those ones